frontend/src/containers/QuestionsList/index.tsx

```diff
@@ -15,7 +16,12 @@ const QuestionsList: FC<QuestionsListProps> = ({questions, isLoading, loadQuesti
     }, [loadQuestions]);
 
     const handleClick = id => {
-        history.push(`question/${id}`);
+        if (questions.map(question => question.id === id &&
```

Comment:
array.map() is always truly so else branch won't be used. Maybe you intended to use array.some or array.every?